cartographer/sensor/compressed_point_cloud.h

```diff
@@ -50,13 +50,18 @@ class CompressedPointCloud {
 
   proto::CompressedPointCloud ToProto() const;
 
+  CompressedPointCloud(const proto::CompressedPointCloud& proto);
+
  private:
   CompressedPointCloud(const std::vector<int32>& point_data, size_t num_points);
 
   std::vector<int32> point_data_;
   size_t num_points_;
 };
 
+CompressedPointCloud ToCompressedPointCloud(
```

Comment:
Remove, since it is no longer needed? Call sites can use the constructor directly.